### DISCUSSION TITLE
Welcome to slider-button-card Discussions!

### DISCUSSION THREAD

**Discussion by mattieha** [mattieha/slider-button-card#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fernando155** [comment]
well, i just want to say... fantastic Job. Really, it looks fantastic.

I just want to give an idea and is just to open this to media. Basically, it would be great that i can select on/off for the tv, and the slide for the volume.

**Reply by mattieha** [reply]
Thanks! I like the idea for media support. I'll add it to the roadmap.

**Reply by mattieha** [reply]
I just released v1.5.0 which adds support for `media_player` entities, please let me know if it works for you!

**Comment by Reynders1** [comment]
Hi Mattieha,
Many thanks for this nice add-on.
While giving it a try, the slider.background doe not seem to be working: I tried all the option (triangle, gradient, ...) but nothing shows, although the slider works. Any idea?
type: grid
cards:
  - type: custom:slider-button-card
    entity: light.lampe_salon
    name: Lampe salon
    show_name: true
    show_state: true
    slider:
      - direction: left-right
      - background: triangle
      - force_square: true

columns: 2
square: true

Many thanks.
Christian

**Comment by Reynders1** [comment]
Hi Mattieha,

With the latest update, the slider no longer works (MacOS BigSure, Safari and Chrome). The slider is no longer changing the value.
Here is the error code I get:
Logger: frontend.js.latest.202106030
Source: components/system_log/__init__.py:190 
First occurred: 9:52:32 (7 occurrences) 
Last logged: 10:08:35

http://homeassistant.local:8123/hacsfiles/slider-button-card/slider-button-card.js:200:19530 TypeError: undefined is not an object (evaluating 't[e[0]]=i')
http://homeassistant.local:8123/hacsfiles/slider-button-card/slider-button-card.js:200:19536 Uncaught TypeError: Cannot set property 'icon' of undefined
:0:0 Script error.

Christian

**Comment by sifpower** [comment]
fantastic job.

You can add control for curtains, which opened and closed two parts at the same time from the center to the sides

**Reply by mattieha** [reply]
@sifpower Something like this?

![CPT2107030018-124x118](https://user-images.githubusercontent.com/3931195/124333373-44274880-db94-11eb-9ab2-e5ad1446c7c3.gif)

**Reply by sifpower** [reply]
yes, very good

**Reply by faizalps501** [reply]
Hi is it possible to make water tank filling animation with this depending on the percentage value from sensor

**Comment by sifpower** [comment]
support for groups?

**Reply by mattieha** [reply]
Groups are not supported (yet), if you want to control multiple lights at the same time use: [Light group](https://www.home-assistant.io/integrations/light.group/l)

**Reply by sifpower** [reply]
I was trying to control all covers on the same button. so i created a group of covers, but it didn't work

**Reply by mattieha** [reply]
Did you use Cover group or regular group?

https://www.home-assistant.io/integrations/cover.group/

**Reply by sifpower** [reply]
yes it's working fine, sorry
My bug

**Comment by sbur83** [comment]
This is Awesome, and the latest update fixed a conflict issue I was having with the love lace slider integration. Cheers mate, bloody legend!

**Reply by mattieha** [reply]
Thanks, glad to hear you like it!

**Comment by davidtheITguy** [comment]
I'm echoing everyone else: this card is awesome. Fantastic job. 

One request for the future, it would be great to be able to pin a minimum slider value > 0. So if someone wanted to start a thermometer temperature at 65 degrees and not 0 for example. Thanks again!

**Comment by Jigsore** [comment]
Sorry for the noob question, I'm new to HA. I've added this through HACS. Rebooted etc. But I can't get it to work. Nothing in the visual editor and if I try to edit the card type I get an error. Any ideas?

**Reply by sbur83** [reply]
Have you added the url to lovelace resources?

**Reply by Jigsore** [reply]
> Have you added the url to lovelace resources?

HACS added it automatically.

**Reply by Jigsore** [reply]
Tried hitting update resource too but still nothing

**Reply by Jigsore** [reply]
Working now. Guess I had to update the resource a couple of times 😁

**Comment by QbaF** [comment]
Great looking and working add-on, congratulation!

I am wonder would it be possible to implement other domains. At this moment I am thinking mainly about `nput_number`. It is available in [lovelace-slider-entity-row](https://github.com/thomasloven/lovelace-slider-entity-row#usage).

PS. It was not obvious to me do You prefer ideas in this thread or as separate Discussion. If it is important, maybe You can write it in the Welcome message.

**Comment by stefanbode** [comment]
I'm on the latest release and have trouble to get up and running. Installed through HACS first. Then also additionally do the manual install. After HACS there was no ressource defined. Anyhow I git the NOT found message if I want to add a custom tile. Any Idea?
![image](https://github.com/mattieha/slider-button-card/assets/16225290/31272e46-28c8-4102-9c26-e01819062833)

**Reply by beerfuzz** [reply]
Reload your interface

**Comment by mdleica** [comment]
Hi Mattieha,
Great pack and awesome design - thank you for that.
I have a plug card that i would want to add a custom field showing the amps. I was having other cards that were working but here it does not show any value. There is any solution for that? 
Another question is if there is any way to to format the weight of the text
Below is the code:
type: custom:slider-button-card
entity: switch.bedroom_ac
slider:
  direction: left-right
  background: gradient
toggle_on_click: true
use_state_color: true
icon:
  use_state_color: true
tap_action:
  action: more-info
show: true
action_button:
  tap_action:
    action: toggle
  mode: custom
  show_spinner: false
  show: true
name: Priza Bedroom AC
custom_fields:
  sensor_value: |
    [[[ 
      var current = states['sensor.bedroom_ac_current']?.state;
      return current && !isNaN(current) ? current + ' A' : 'N/A'; 
    ]]]
styles:
  custom_fields:
    sensor_value:
      - position: absolute
      - bottom: 12%
      - right: 5%
      - font-size: 12px
      - font-weight: bold
      - color: |
          [[[ 
            var current = parseFloat(states['sensor.bedroom_ac_current']?.state);
            return current > 0.12 ? 'red' : 'blue'; 
          ]]]
compact: false

Thank you
